app/views/form_drafts/show.haml

```diff
@@ -43,6 +43,6 @@
 
 .form
   .actions
-    = button_to 'Continue editing', edit_form_draft_url(@draft), method: :get
+    = button_to 'Continue editing', edit_form_draft_path(@draft), method: :get
```

Comment:
Yup, I didn't know the difference at the time and thought it was broken because of that. Ended up fixing the real problem and then the fact that I had changed that bit.. well, I forgot about it entirely because it still works. Change to whatever you like.